### PR TITLE
CCXDEV-7142 update the Insights overview

### PR DESCRIPTION
**Pull request by xJustin**
This updates the Insights Advisor overview with an updated description. Disregard the outdated procedures below as they will be updated in another PR. 

Versions: enterprise-4.7, enterprise-4.8, enterprise-4.9, enterprise-4.10

Preview:
https://deploy-preview-41679--osdocs.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.html 

Jira:
https://issues.redhat.com/browse/CCXDEV-7142

SME: radekvokal , CCX-presentation 
QE: ikerreyes

### CODE REVIEW

File: modules/insights-operator-advisor-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_content-type: CONCEPT
+[id="insights-operator-advisor-overview_{context}"]
+= About Red Hat Insights Advisor for {product-title}
+
+You can use Insights Advisor to assess and monitor the health of your {product-title} clusters. Whether you are concerned individual clusters, or with your whole infrastructure, it is important to be aware of your exposure to issues that can affect service availability, fault tolerance, performance, or security.
```
Comment (radekvokal) [inline modules/insights-operator-advisor-overview.adoc:9]: `whether you are concerned about`?

File: modules/insights-operator-advisor-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Insights repeatedly analyzes the data Insights Operator sends using a database of *Recommendations*, which are sets of conditions that can leave your {product-title} clusters at risk. Your data is then uploaded to the Insights Advisor page on Red Hat Hybrid Cloud Console where you can perform the following actions:
```
Comment (radekvokal) [inline modules/insights-operator-advisor-overview.adoc:11]: Insights Advisor service or app (not just page)

File: modules/insights-operator-advisor-overview.adoc (hunk 1 above)
Comment (radekvokal) [inline modules/insights-operator-advisor-overview.adoc:9]: `concerned about`?

File: modules/insights-operator-advisor-overview.adoc (hunk 2 above)
Comment (radekvokal) [inline modules/insights-operator-advisor-overview.adoc:11]: `Advisor service` or app (not `page`)

File: modules/insights-operator-advisor-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+* See all of the recommendations for your clusters.
```
Comment (radekvokal) [inline modules/insights-operator-advisor-overview.adoc:13]: * See clusters impacted by a specific recommendation

File: modules/insights-operator-advisor-recommendations.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_content-type: CONCEPT
+[id="insights-operator-advisor-recommendations_{context}"]
+= Understanding Insights Advisor recommendations
+
+Insights Advisor bundles information about known configuration issues that can negatively affect the service availability, fault tolerance, performance, or security of your clusters. This information set is called a recommendation in Insights Advisor and includes the following information:
```
Comment (radekvokal) [inline modules/insights-operator-advisor-recommendations.adoc:9]: s/ known configuration issues/various clusters states and component configuration

**Comment by xJustin** [comment]
Thanks @radekvokal ! All changes incorporated.

**Comment by ikerreyes** [comment]
lgtm

File: modules/insights-operator-advisor-overview.adoc [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_content-type: CONCEPT
+[id="insights-operator-advisor-overview_{context}"]
+= About Red Hat Insights Advisor for {product-title}
+
+You can use Insights Advisor to assess and monitor the health of your {product-title} clusters. Whether you are concerned about individual clusters, or with your whole infrastructure, it is important to be aware of your exposure to issues that can affect service availability, fault tolerance, performance, or security.
+
+Insights repeatedly analyzes the data Insights Operator sends using a database of *Recommendations*, which are sets of conditions that can leave your {product-title} clusters at risk. Your data is then uploaded to the Insights Advisor service on Red Hat Hybrid Cloud Console where you can perform the following actions:
```
Comment (bobfuru) [inline modules/insights-operator-advisor-overview.adoc:11]: Using "that" in this case can improve readability; when at first I read it, I was trying to figure out if "data" was part of the Operator name with missed casing (my mind can wander).
```suggestion
Insights repeatedly analyzes the data that Insights Operator sends using a database of *Recommendations*, which are sets of conditions that can leave your {product-title} clusters at risk. Your data is then uploaded to the Insights Advisor service on Red Hat Hybrid Cloud Console where you can perform the following actions:
```

Also wondering about the bolding of Recommendations. I'm unfamiliar, so if it is part of the console UI then it makes sense, but if it is introducing a concept, you might consider underscores instead (as suggested in our OpenShift docs [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)). Also, if not a UI element, consider using lowercase "recommendations" to match casing in the next module.

File: modules/insights-operator-advisor-recommendations.adoc [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+:_content-type: CONCEPT
+[id="insights-operator-advisor-recommendations_{context}"]
+= Understanding Insights Advisor recommendations
+
+Insights Advisor bundles information about various cluster states and component configurations that can negatively affect the service availability, fault tolerance, performance, or security of your clusters. This information set is called a recommendation in Insights Advisor and includes the following information:
+
+* *Name:* A concise description of the recommendation
+* *Added:* When the recommendation was published to the Insights Advisor archive
+* *Category* Whether the issue has the potential to negatively affect service availability, fault tolerance, performance, or security
```
Comment (bobfuru) [inline modules/insights-operator-advisor-recommendations.adoc:13]: Missing a colon

**Review by bobfuru — COMMENTED** [review]
Couple of comments but overall this LGTM!

File: modules/insights-operator-advisor-overview.adoc (hunk 5 above)
Comment (xJustin) [reply modules/insights-operator-advisor-overview.adoc:11]: It is a UI element, but the way I word it here is more conceptual. I will just unbold it.

**Comment by xJustin** [comment]
Thanks @bobfuru ! I incorporated all of your comments. Decided to go with underscores on `recommendations` as it is the first mention of a new term per the style guide. 

This PR is good to merge and try to cherry-pick whenever you get the chance.

**Comment by bobfuru** [comment]
LGTM

**Pull request merged**